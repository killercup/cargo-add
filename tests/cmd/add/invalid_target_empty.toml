bin.name = "cargo-add"
args = ["add", "my-package", "--target", ""]
status.code = 2
stdout = ""
stderr = """
error: The argument '--target <TARGET>' requires a value but none was supplied

USAGE:
    cargo add [OPTIONS] <DEP_ID>...

For more information try --help
"""
fs.sandbox = true

[env.add]
CARGO_IS_TEST="1"
