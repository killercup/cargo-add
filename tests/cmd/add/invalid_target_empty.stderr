error: The argument '--target <TARGET>' requires a value but none was supplied

USAGE:
    cargo add [OPTIONS] <CRATE>...

For more information try --help
